### ISSUE TITLE
Segmentation Fault when executing collect.py

### ISSUE THREAD

**Issue by wang414** [ZiwenZhuang/parkour#44]
Thanks for your excellent work. I am facing segmentation fault while running ``python legged_gym/scripts/collect.py --headless --task a1_distill``.  I traced back for the problem, and found that it was caused by the forward_depth observation when the code is creating for the environment. Do you have any idea about it. 

My GPU is A100-40G, pytorch version is 1.10.0+cu113.

**Comment by AlorithmKing** [comment]
i meet the same issue when i run  python legged_gym/scripts/train.py --task a1_leap

Building extension module gymtorch...
Allowing ninja to set a default number of workers... (overridable by setting the environment variable MAX_JOBS=N)
ninja: no work to do.
Loading extension module gymtorch...
Setting seed: 1
Using LeggedRobotField.__init__, num_obs and num_privileged_obs will be computed instead of assigned.
Not connected to PVD
+++ Using GPU PhysX
Physics Engine: PhysX
Physics Device: cuda:0
GPU Pipeline: enabled
Segmentation fault (core dumped)

**Comment by ZiwenZhuang** [comment]
> i meet the same issue when i run python legged_gym/scripts/train.py --task a1_leap
> 
> Building extension module gymtorch... Allowing ninja to set a default number of workers... (overridable by setting the environment variable MAX_JOBS=N) ninja: no work to do. Loading extension module gymtorch... Setting seed: 1 Using LeggedRobotField.**init**, num_obs and num_privileged_obs will be computed instead of assigned. Not connected to PVD +++ Using GPU PhysX Physics Engine: PhysX Physics Device: cuda:0 GPU Pipeline: enabled Segmentation fault (core dumped) My GPU is 4090 torch==2.0.0+cu118 torchvision==0.15.0+cu118 torchaudio==2.0.0+cu118

It seems you forgot to add `--headless` argument in a headless server?

**Comment by ZiwenZhuang** [comment]
> Thanks for your excellent work. I am facing segmentation fault while running `python legged_gym/scripts/collect.py --headless --task a1_distill`. I traced back for the problem, and found that it was caused by the forward_depth observation when the code is creating for the environment. Do you have any idea about it.
> 
> My GPU is A100-40G, pytorch version is 1.10.0+cu113.

If it is the depth rendering problem, please make sure `*Cfg.sim.no_camera` is set to `False` in your config file.

**Comment by shiluhe** [comment]
> Thanks for your excellent work. I am facing segmentation fault while running `python legged_gym/scripts/collect.py --headless --task a1_distill`. I traced back for the problem, and found that it was caused by the forward_depth observation when the code is creating for the environment. Do you have any idea about it.感谢您出色的工作。我在运行 `python legged_gym/scripts/collect.py --headless --task a1_distill` 时遇到了段错误。我追踪了问题，发现是由代码创建环境时对 forward_depth 的观测引起的。您对此有任何想法吗？
> 
> My GPU is A100-40G, pytorch version is 1.10.0+cu113.我的 GPU 是 A100-40G，pytorch 版本是 1.10.0+cu113。

Have you solved this problem? I meet the problem too
